src/internal/views/nunjucks/billing/two-part-tariff-ready.njk

```diff
@@ -39,9 +39,9 @@
 <section class="govuk-!-margin-bottom-9">
   <h2 class="govuk-heading-l" id="issuesTitle">
     {% if totals.ready == 1 %}
```

Comment:
Should we try to accommodate zero licences?

Reply:
The link to this page (located on the review page) is only visible if `totals.ready` is greater than 0